package.json

```diff
@@ -66,5 +68,10 @@
     "collectCoverageFrom": [
       "src/{app,helpers}/**/*.{ts,tsx}"
     ]
+  },
+  "babel": {
```

Comment:
I added it because the `yarn run test:browser` needs to know how to compile typescript files and `react-app` contains that logic. If there is a way to run `jest-puppeteer` with `react-scripts test` then this would not be needed